Git process.md

```diff
@@ -29,3 +29,69 @@ While the application is still not deployed on a production server, you can omit
 **Make small commits, following the Single Responsibility Principle in Git.** This makes commits easier to review when making pull requests, and it's easier to notice what's going on when something goes wrong.
 
 **Don't use `git add .`** Review what you're adding to your repo—this is the #1 cause of making unwanted changes.
+
+### Github Pull Request Descriptions
+
+It's essential to write a good Pull Request description. Reviewers will usually read it before
+looking at the diff, so make sure it gives them enough context, so they know what they are looking at.
+
+Another purpose of the good description is to provide the documentation for a future reference.
+A new developer might join the project in the future. The description will help him to understand
+the motivation behind the implementation of a specific feature.    
```

Comment:
```suggestion
the motivation behind the implementation of a specific feature.    

Taking this a step forward, writing good commit descriptions has an added benefit of granulating big pull request descriptions into logical per-file chunks, which can also be inspected when annotating files. Since GitHub is smart enough to automatically include commit comments at the top of the PR description, it is easy enough to merge and move the comments to the correct section and edit if needed.
```

Reply:
I suggest you create another PR where you can address the topic of commit messages.